docs/helpers/ViewportBehavior.md

```diff
@@ -0,0 +1,69 @@
+---
+title: ViewportBehavior class
+author: h82258652
+description: This behavior allows you to listen an element enter or exit the ScrollViewer viewport
+keywords: windows 10, uwp, windows community toolkit, uwp community toolkit, uwp toolkit, your control name
+dev_langs:
+  - csharp
+---
+
+# ViewportBehavior
+
+This behavior allows you to listen an element enter or exit the ScrollViewer viewport.
+
+## Syntax
+
+```xaml
+<Page 
+    xmlns:interactivity="using:Microsoft.Xaml.Interactivity"
+    xmlns:behaviors="using:Microsoft.Toolkit.Uwp.UI.Animations.Behaviors" />
+
+<ScrollViewer>
+    <StackPanel Orientation="Vertical"
+                Height="3000">
+        <Border Width="100"
+                Height="100">
+            <interactivity:Interaction.Behaviors>
+                <behaviors:ViewportBehavior x:Name="ViewportBehavior"
+                                            EnteringViewport="Border_EnteringViewport"
+                                            EnteredViewport="Border_EnteredViewport"
+                                            ExitingViewport="Border_ExitingViewport"
+                                            ExitedViewport="Border_ExitedViewport" />
+            </interactivity:Interaction.Behaviors>
+        </Border>
+    </StackPanel>
+</ScrollViewer>
+```
+
+## Properties
+
+| Property | Type | Description |
+| -- | -- | -- |
+| IsInViewport | bool | Gets a value indicating whether associated element is in the ScrollViewer viewport |
+| IsFullyInViewport | bool | Gets a value indicating whether associated element is fully in the ScrollViewer viewport |
+| IsRemoveBehaviorAfterEnteredViewport | bool | Gets or sets a value indicating whether remove this behavior after associated element entered viewport |
```

Comment:
```suggestion
| IsAlwaysOn | bool | Gets or sets a value indicating whether this behavior will remain attached after the associated element enters the viewport. By default, when false, the behavior will remove itself after the attached element enters. |
```

Reply:
@h82258652 small tweak here to match the discussion and property update on the main PR which I've just merged.  Sorry again for the delay.  If you can just accept this suggestion, we'll be good to merge the doc PR in too!  Thanks again for this contribution!

Reply:
@h82258652 are you able to confirm this change for us so we can merge the docs in? Thanks!

Reply:
@JustinXinLiu have you heard lately from @h82258652?

Reply:
@michael-hawker Done. Sorry for takes so long.

Reply:
No worries, thanks for making the change!